### PR TITLE
Group statistics: replace Integer by Long types

### LINKED ISSUE #595: Group statistics: Integer overflow if one field is higher than 2^31
Hello,

I'm using gitlab API to get groups statistics. It seems like the group statistics class fields use Integer types instead of Long and won't be able to handle numbers 2^31. I see that in the project statistics class, you are already using Long. I'll open a PR to update the group statistics class as well.

### PR DESCRIPTION
**Pull request by BlackYoup**
Fixes #595

I've tested this on my project and it can now handle 2^31 storage.

### CODE REVIEW

File: src/main/java/org/gitlab4j/api/models/Group.java [hunk 1]
```diff
@@ -11,40 +11,40 @@
...
 
     public class Statistics {
-        private Integer storageSize;
-        private Integer repositorySize;
-        private Integer lfsObjectsSize;
-        private Integer jobArtifactsSize;
+        private long storageSize;
+        private long repositorySize;
+        private long lfsObjectsSize;
+        private long jobArtifactsSize;
```
Comment (gmessner) [inline src/main/java/org/gitlab4j/api/models/Group.java:17]: Change ```long``` to ```Long```

**Comment by BlackYoup** [comment]
Thanks for your review! I updated the PR

**Comment by gmessner** [comment]
@BlackYoup Thanks for the quick turn-around.  Will merge today ad will include in the next release scheduled n 7/25/2020.

**Comment by BlackYoup** [comment]
You're welcome, thank you very much :+1:

**Comment by gmessner** [comment]
@BlackYoup GitLab4J-API Release 4.15.3 has been published and is now available.

**Pull request merged**